src/test/javascript/spec/component/exam/exam-timer.component.spec.ts

```diff
@@ -40,20 +40,20 @@ describe('ExamTimerComponent', () => {
 
     it('should update display times', () => {
         let duration = dayjs.duration(15, 'minutes');
-        expect(component.updateDisplayTime(duration)).toBe('15:00');
+        expect(component.updateDisplayTime(duration)).toBe('15min');
         duration = dayjs.duration(-15, 'seconds');
-        expect(component.updateDisplayTime(duration)).toBe('00');
+        expect(component.updateDisplayTime(duration)).toBe('0min 0s');
         duration = dayjs.duration(8, 'minutes');
-        expect(component.updateDisplayTime(duration)).toBe('08:00');
+        expect(component.updateDisplayTime(duration)).toBe('8min 0s');
         duration = dayjs.duration(45, 'seconds');
-        expect(component.updateDisplayTime(duration)).toBe('45');
+        expect(component.updateDisplayTime(duration)).toBe('0min 45s');
```

Comment:
If its 0min, should the minutes part just be omitted?

Reply:
I think this can be discussed if we should display it or not. But since Stephan wants to have old exam mode timer, old version had this format. (0min 45s) 
<img width="1512" alt="Screenshot 2024-07-17 at 16 05 17" src="https://github.com/user-attachments/assets/157fc38b-33de-4269-bc83-540e17cff81a">

Reply:
I talked to Stephan now and he has the opinion to keep 0min